### PR TITLE
Revert [Stratconn 3257] Add batching to segment connections (#1859)

### PR DESCRIPTION
**Pull request by Innovative-GauravKochar**
Reverts https://github.com/segmentio/action-destinations/pull/1859
This Pull request is to to remove Segment Connections batching changes.
Jira ticket:- https://segment.atlassian.net/browse/STRATCONN-3716

Interrelated PRs:- https://github.com/segmentio/integrations-go/pull/419
https://github.com/segmentio/integrations/pull/2925
 
## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/core/src/destination-kit/index.ts [hunk 1]
```diff
@@ -603,7 +603,7 @@ export class Destination<Settings = JSONObject, AudienceSettings = JSONObject> {
       audienceSettings = events[0].context?.personas?.audience_settings as AudienceSettings
     }
 
-    return action.executeBatch({
+    await action.executeBatch({
```
Comment (varadarajan-tw) [inline packages/core/src/destination-kit/index.ts:606]: this revert to actions-core might break few integration test cases. Do you have the PR to fix those tests in integrations?

**Review by varadarajan-tw — COMMENTED** [review]
unit tests seem to be failing. Could you take a look into it?

**Comment by Innovative-GauravKochar** [comment]
> return action.executeBatch({
>     await action.executeBatch({

yes ,I am on it

File: packages/core/src/destination-kit/index.ts (hunk 1 above)
Comment (Innovative-GauravKochar) [reply packages/core/src/destination-kit/index.ts:606]: Yes , I am on it

Comment (Innovative-GauravKochar) [reply packages/core/src/destination-kit/index.ts:606]: @varadarajan-tw  please review integration's PR https://github.com/segmentio/integrations/pull/2925

**Review by varadarajan-tw — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @Innovative-GauravKochar PR deployed.

**Pull request merged**